### PR TITLE
DA-3331: org affiliation changes switch list org to use platform service - Part 1

### PR DESCRIPTION
**Pull request by ajinkyan83**
Jira Ticket: https://jira.linuxfoundation.org/browse/DA-3331

Below 2 APIs are updated as per Case 1 mentioned here - https://jira.linuxfoundation.org/browse/DA-3331?focusedCommentId=410087&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-410087

`list_organizations`
`add_enrollment`

For e.g. 

http://127.0.1.1:8080/v1/affiliation/project1/list_organizations?q=linux%20foundation&page=1&rows=10 now returns data using org service 
```

{
    "organizations": [
        {
            "domains": [],
            "id": "0016s000006UKKqAAO",
            "name": "The Linux Foundation"
        }
    ],
    "page": 1,
    "rows": 1,
    "scope": "project1",
    "search": "q=The Linux Foundation ",
    "user": "ajinkyan"
}

```

Also verified and tested add enrollment API locally using test env vars - http://127.0.1.1:8080/v1/affiliation/project1/add_enrollment/... 

A new env variable is introduced `PLATFORM_ORG_SERVICE_ENDPOINT` which points to `https://api-gw.dev.platform.linuxfoundation.org/organization-service/v1` for dev endpoint.

In case the org does not exist both in affiliations db and org service (SFDC), I am returning an error for now as shown below -

`{"error":"LookupOrganization: Organization: 'The Linux Foundadadadadadation Test3' not found in db as well org service","level":"error","msg":"PostAddEnrollmentHandlerFunc(error): Request IP: 10.0.2.2:62351, Request Agent: PostmanRuntime/7.26.8, method: POST, path: /v1/affiliation/project1/add_enrollment/fa880b65f6af233ff3be9cf0ed8d2cd307ac1ad5/The Linux Foundadadadadadation Test3","time":"2021-01-25T17:47:03+05:30"}`

Related PR: https://github.com/LF-Engineering/dev-analytics-libraries/pull/21

### CODE REVIEW

File: platform/organization.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+// New return ES connection
+func New(client *orgs.Org) Service {
+	return &service{
+		client: client,
+	}
+}
+
+func (s *service) GetListOrganizations(q string, rows, page int64) (*models.GetListOrganizationsServiceOutput, error) {
```
Comment (enkhalifapro) [inline platform/organization.go:32]: Just comment for public function

**Review by enkhalifapro — APPROVED** [review]

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by lukaszgryglicki** [comment]
@ajinkyan83 can you please send me all new ENV variables values?
I mean those:
```
		os.Getenv("PLATFORM_ORG_SERVICE_ENDPOINT"),
 		os.Getenv("ELASTIC_CACHE_URL"),
 		os.Getenv("ELASTIC_CACHE_USERNAME"),
 		os.Getenv("ELASTIC_CACHE_PASSWORD"),
 		os.Getenv("STAGE"),
 		os.Getenv("AUTH0_PROD_GRANT_TYPE"),
 		os.Getenv("AUTH0_PROD_CLIENT_ID"),
 		os.Getenv("AUTH0_PROD_CLIENT_SECRET"),
 		os.Getenv("AUTH0_PROD_AUDIENCE"),
 		os.Getenv("AUTH0_TOKEN_ENDPOINT"),
```
I need values for all of them for `test`, `prod` (and eventually/optionally `local`) envs, so when this PR is merged I can still build and deploy API locally.
The convention is to have all of them saved as `*.secret` files in `./helm/da-affiliation/secrets/`(so they're not checked in to therepo/.gitignored):
Fo example there should be 
- `./helm/da-affiliation/secrets/STAGE.test.secret`, `./helm/da-affiliation/secrets/STAGE.prod.secret` files and so on for all new ENV variables/secrets.

Ideally send me a ZIP file containing all those 20 files on slack (10 env vars x 2 environments test/prod).

Thanks!

**Comment by ajinkyan83** [comment]
@lukaszgryglicki as suggested above, I sent the new env vars' secret files over slack.

**Comment by lukaszgryglicki** [comment]
Got them, will update when I can
You should also put them in K8s/Helm secrets template: https://github.com/LF-Engineering/dev-analytics-affiliation/blob/master/helm/da-affiliation/templates/secrets.yaml

**Comment by ajinkyan83** [comment]
> Got them, will update when I can
> You should also put them in K8s/Helm secrets template: https://github.com/LF-Engineering/dev-analytics-affiliation/blob/master/helm/da-affiliation/templates/secrets.yaml

Done. As discussed, also updated https://github.com/LF-Engineering/dev-analytics-affiliation/blob/master/helm/da-affiliation/templates/api.yaml

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**